common/src/main/java/io/netty/util/NetUtil.java

```diff
@@ -925,7 +859,7 @@ public static Inet6Address getByName(CharSequence ip, boolean ipv4Mapped) {
             }
         }
 
-        if (ipv4Mapped && ipv4Seperators > 0 &&
+        if (ipv4Mapped && ipv4Separators > 0 &&
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of conditional operators (11) used in the expression (maximum allowed 5). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1067)